buddyboss/buddyboss-platform
# PROD-708#3095 fix filtered moderated users from the bp_get_followers and bp_get_following functions

Added function and filters to exclude moderated users from the widget earlier the count was also counting the moderated(hidden users) in the following functions:
bp_get_followers()
bp_get_following()

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #3095 .

### How to test the changes in this Pull Request:

Steps to reproduce the behavior:

1. Enable the Moderation Component
2. And allow members to block each other (member blocking) and enable the auto suspend on 1 time blocking from the Moderation Settings( Buddyboss > Settings > Moderation )
3. Login as few multiple members with View As and click Follow
4. Get back to the admin account and add the (BB) Members Following Me Widget to the Activity Page Sidebar
5. Check the follower count in widget should be equal to the numbers of members avatar is showing in the widget
6. Now suspend or block the user by going to their profile page and clicking on block and check

### Proof Screenshots or Video
https://somup.com/crXZX20YDi

### Other information:




### Changelog entry

> Added function and filters to exclude moderated users from the bp_get_followers and bp_get_following functions
